File: roligamalaren/submissions/accepted/joshua.cpp
#include <bits/stdc++.h>

using namespace std;


int main()
{
	int r, c;
	cin >> r >> c;

	vector<string> start(r);
	vector<string> goal(r);

	for (int i = 0; i < r; i++) cin >> start[i];
	for (int i = 0; i < r; i++) cin >> goal[i];

	int differences = 0;
	for (int i = 0; i < r; i++) for (int j = 0; j < c; j++) if (start[i][j] != goal[i][j]) differences++;

	cout << differences << "\n";

	return 0;
}

File: denroligamalaren/submissions/accepted/joshua.py
#!/usr/bin/python3

r,c = [int(i) for i in input().split()]

start = []
goal = []
for i in range(r):
    start.append(input())
for i in range(r):
    goal.append(input())

differences = 0
for i in range(r):
    for j in range(c):
        if start[i][j] != goal[i][j]:
            differences += 1

print(differences)
